### PR TITLE
github workflow: drop measure test flakiness 

### LINKED ISSUE #17774: Drop Measure Test Flakiness
### What would you like to be added?

https://github.com/etcd-io/etcd/actions/workflows/measure-test-flakiness.yaml consistently fails. 

Don't think we should fix it as we building a replacement.

Before removal we should ensure that replacement is working correctly and runs every day instead of once per week.

### Why is this needed?

We are migrating to https://github.com/etcd-io/etcd/actions/workflows/measure-testgrid-flakiness.yaml

### PR DESCRIPTION
**Pull request by jimmy-bro**
Resolve #17774

### CODE REVIEW

**Comment by ivanvc** [comment]
Hi @jimmy-bro, thanks for your contribution. Could you please sign your commit? So the developer certificate of origin (DCO) check passes, i.e:

```
git rebase HEAD~1 --signoff
git push --force
```

Thanks!

Ref: https://github.com/etcd-io/etcd/blob/main/CONTRIBUTING.md#commit-your-change

**Review by serathius — APPROVED** [review]

**Comment by serathius** [comment]
cc @siyuanfoundation

File: .github/workflows/measure-test-flakiness.yaml
Comment (stackbaek) [inline .github/workflows/measure-test-flakiness.yaml:1]: Can we also delete `scripts/measure-test-flakiness.sh` as it is not needed any longer?

cc: @serathius

Comment (jimmy-bro) [reply .github/workflows/measure-test-flakiness.yaml:1]: Good idea,  we are no longer need` scripts/measure-test-flakiness.sh`

**Comment by ahrtr** [comment]
Eventually we also need to remove the Makefile target `bin/etcd-test-analyzer`

**Comment by ahrtr** [comment]
Please squash the commits.

**Comment by jimmy-bro** [comment]
Thank you for the reminder, @ahrtr . It has now been squashed into a single commit.

**Review by siyuanfoundation — APPROVED** [review]

**Comment by siyuanfoundation** [comment]
LGTM. Thanks @jimmy-bro !

**Review by ahrtr — APPROVED** [review]

**Pull request merged**